InvenTree/InvenTree/management/commands/rebuild_models.py

```diff
@@ -27,6 +28,7 @@ def handle(self, *args, **kwargs):
         try:
             print("Rebuilding PartCategory objects")
 
+            # InvenTree imports
```

Comment:
Can we disable these messages appearing throughout the code?

Reply:
Yes, they can be configured